### DISCUSSION TITLE
Error when installing

### DISCUSSION THREAD

**Discussion by HassanAlam** [AI4Finance-Foundation/FinRL#858] [Q&A]
Hi,

I get this error when I try to install FinRL

ERROR: Could not find a version that satisfies the requirement ray[default,tune]==1.3.0 (from finrl) (from versions: 1.4.1, 1.5.0, 1.5.1, 1.5.2, 1.6.0, 1.7.0rc0, 1.7.0, 1.7.1, 1.8.0, 1.9.0rc1, 1.9.0rc2, 1.9.0, 1.9.1rc0, 1.9.1, 1.9.2, 1.10.0rc0, 1.10.0, 1.11.0rc0, 1.11.0rc1, 1.11.0, 1.11.1, 1.12.0rc1, 1.12.0, 1.12.1, 1.13.0, 2.0.0rc0, 2.0.0rc1, 2.0.0, 2.0.1, 2.1.0, 2.2.0)
ERROR: No matching distribution found for ray[default,tune]==1.3.0

**Comment by zhumingpassional** [comment]
pls download .zip, or git clone
